web/static/js/decls/jquery.js

```diff
@@ -1 +1,3 @@
-declare var $: Function
+declare var $: Function;
+declare var it: Function;
```

Comment:
Create a new `mocha.js` file in this same directory and move those two declarations there.